test/GameLogic/e2e/Iceblock.test.ts

```diff
@@ -12,7 +12,7 @@ const {getX, getY} = Direction8Utils;
 describe('GameLogic.e2e - Iceblock', () => {
 	it('Iceblock should not move initiially', () => {
 		TestLevelBuilder.newLevel()
-			.addEntity(new Iceblock(new Pushable()), 5, 5)
+			.addEntity(new Iceblock(undefined), 5, 5)
```

Comment:
You can remove `undefined` altogether, not passing an argument is like passing `undefined`. Ditto for all other places where this occurs

Reply:
Typescript says no

Reply:
nvm, Typescript says yes if undefined is added as a default to the constructor parameter